source/nRF5xServiceDiscovery.h

```diff
@@ -300,6 +321,8 @@ class nRF5xServiceDiscovery : public ServiceDiscovery
     CharUUIDDiscoveryQueue      charUUIDDiscoveryQueue;
 
     TerminationCallback_t       onTerminationCallback;
+
+    nRF5xDiscoveredCharacteristic remainingCharacteristic;
```

Comment:
what is this `remainingCharacteristic`? It seems to be key to the new implementation. I would have liked to find a comment describing its function.